### ISSUE TITLE
BTC regnet test locktime error

### ISSUE THREAD

**Issue by buck54321** [decred/dcrdex#164]
```
--- FAIL: TestWallet (1.53s)
    regnet_test.go:332: refund error: -26: non-final (code 64)

```

This only occurs if regnet has been running idle for a day or two. The error indicates that the locktime requirement has not been met. I've observed this only twice. Mining a couple blocks before testing does not help. Lock time for the swap output is set in the past during that particular test, and it passes on a freshly started simnet harness. It seems to also pass if I reload the `gamma` wallet by sending it more BTC from `beta` and mine a block. 

To reproduce, start the simnet harness, wait a day, and then run `go test -v -tags harness` fromm **client/assets/btc**

**Comment by itswisdomagain** [comment]
From discussion in chat:
For btc, the locktime is checked against the median time of the last 11 blocks rather than the current time of refund attempt.

Helpful links:
https://github.com/bitcoin/bitcoin/blob/668de70be039a4f1ffcf20aeae2a22ee71fc55a8/src/validation.cpp#L236
https://github.com/bitcoin/bitcoin/blob/452bb90c718da18a79bfad50ff9b7d1c8f1b4aa3/doc/release-notes/release-notes-0.12.1.md#bip113-locktime-enforcement-soft-fork

**Comment by buck54321** [comment]
Presumably fixed in #401.

**Comment by itswisdomagain** [comment]
A more thorough fix would require mining some blocks to ensure the timelock is expired, something like what's done in #402:
https://github.com/decred/dcrdex/blob/27bdd9481eed5ec0ee258f228ee27aa1d459f09f/client/core/trade_simnet_test.go#L536-L544

Since the locktime in the test is 24 hours before the current time, the median time of the last 11 blocks should be well above that, unless no blocks have been recently mined, such as `if regnet has been running idle for a day or two` as mentioned in the original issue description.
